plugins/dell-kestrel/fu-dell-kestrel-ec.c

```diff
@@ -455,7 +455,7 @@ fu_dell_kestrel_ec_run_passive_update(FuDellKestrelEc *self, GError **error)
 	/* ec included in cmd, set bit2 in data for tbt */
 	fu_byte_array_append_uint8(req, FU_DELL_KESTREL_EC_HID_CMD_SET_PASSIVE);
 	fu_byte_array_append_uint8(req, 1); // length of data
-	fu_byte_array_append_uint8(req, 0x02);
+	fu_byte_array_append_uint8(req, 0x07);
```

Comment:
can you add this as a rustgen struct? I think it would be a lot clearer for future people!

Reply:
just checked and this is the only device in the bitmap , the rest are unused. Maybe make it a `#define`?

Reply:
Nah, I think we can get rid of some of those fu_byte_array_append_uint8()'s